skada/deep/_graph_alignment.py

```diff
@@ -87,19 +91,28 @@ def forward(
         )
 
         # update classification function
-        loss_adv = self.domain_criterion_(
-            domain_pred_s, domain_label
-        ) + self.domain_criterion_(domain_pred_t, domain_label_target)
+        loss_adv = (
+            self.reg_adv
+            * eff
```

Comment:
We did not put a scheduler in other adversarial methods. I think you should remove it to be consistent.